### ISSUE TITLE
Scrolling container located outside of component

### ISSUE THREAD

**Issue by plcarmel** [offirgolan/ember-light-table#365]
It should be possible to scroll the table even if the intimate details of the scrolling process is not handled by `ember-light-table` itself.

Here's my use case:
I have a dashboard in my app where I have a navbar on the left and a content pages on the right.  Scrolling of the content pages is already handled by the dashboard component itself.  However, I need an infinite scrolling like the one already offered in `ember-light-table`.

**Comment by alexander-alvarez** [comment]
Can you maybe rephrase or describe with images?

What you're saying should be able to be done leveraging the `scrollToY` nature of the underlying `ember-scrollable`.

Are you talking about having a scrollbar not tied to the table that scrolls the content of the table? If so you can take it by having them have the same size content and scrolling them in sync as shown in the cookbook section of `ember-scrollable`

**Comment by plcarmel** [comment]
**_Are you talking about having a scrollbar not tied to the table that scrolls the content of the table?_**
Yes, exactly

**_If so you can take it by having them have the same size content and scrolling them in sync as shown in the cookbook section of `ember-scrollable`_**
Thank you for pointing that out. And I think this is going to work even they are not the same size, as long as I take the offset into account.

**Comment by alexander-alvarez** [comment]
> And I think this is going to work even they are not the same size, as long as I take the offset into account.

yup. the `scrollToY` lets you take advantage of that, you will just need to manage those ratio's and state somewhere else.
